### PR TITLE
[SOC] Translate the dev-run page

### PR DESCRIPTION
**Pull request by QuakeWang**
Translate the [dev-run](https://dolphinscheduler.apache.org/zh-cn/docs/dev/user_doc/dev_run.html) page.

### CODE REVIEW

**Comment by QuakeWang** [comment]
@ptyp PTAL

File: docs/en-us/dev/user_doc/dev-run.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+## Development Environment Setup
+
+>    Reference: [DolphinScheduler builds the development environment on Windows local.](https://dolphinscheduler.apache.org/zh-cn/blog/DS_run_in_windows.html)
+
+#### 1. Download source code
```
Comment (ptyp) [inline docs/en-us/dev/user_doc/dev-run.md:5]: "Download the source code"

File: docs/en-us/dev/user_doc/dev-run.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+```shell
+mkdir dolphinscheduler
+cd dolphinscheduler
+git clone git@github.com:apache/dolphinscheduler.git
+```
+
+We use the dev branch here.
+
+#### 2. Windows install of zookeeper.
```
Comment (ptyp) [inline docs/en-us/dev/user_doc/dev-run.md:17]: "The zookeeper installation for Windows"

File: docs/en-us/dev/user_doc/dev-run.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+iii. Create new zkData, zkLog folders in zk's directory.
+
+iv. Copy the zoo_sample.cfg file from the conf directory. Then rename it to zoo.cfg and change the configuration of the data and logs in it. For example: 
+
+```
+dataDir=D:\\code\\apache-zookeeper-3.6.3-bin\\zkData
+dataLogDir=D:\\code\\apache-zookeeper-3.6.3-bin\\zkLog
+```
+
+v. Run zkServer.cmd in the bin, and then run zkCli.cmd to view the running status of zk. You can view the zk node information, which means the installation is successful.
```
Comment (ptyp) [inline docs/en-us/dev/user_doc/dev-run.md:32]: "If you can view ..., it means the ..."

File: docs/en-us/dev/user_doc/dev-run.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+```properties
+#registry.plugin.dir config the Registry Plugin dir.
+registry.plugin.dir=./dolphinscheduler-dist/target/dolphinscheduler-dist-1.3.6-SNAPSHOT/lib/plugin/registry/zookeeper
+
+registry.plugin.name=zookeeper
+registry.servers=127.0.0.1:2181
+```
+
+vii. Add console output to logback-worker.xml, logback-master.xml, and logback-api.xml.
```
Comment (ptyp) [inline docs/en-us/dev/user_doc/dev-run.md:64]: "Add the console output"

File: docs/en-us/dev/user_doc/dev-run.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+```xml
+<root level="INFO">
+    <appender-ref ref="STDOUT"/>  <!-- 添加控制台输出 -->
```
Comment (ptyp) [inline docs/en-us/dev/user_doc/dev-run.md:68]: The Chinese sentence (添加控制台输出) is forgotten to be translated.

File: docs/en-us/dev/user_doc/dev-run.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+    <appender-ref ref="APILOGFILE"/>
+    <appender-ref ref="SKYWALKING-LOG"/>
+</root>
+```
+
+viii. Start MasterServer
```
Comment (ptyp) [inline docs/en-us/dev/user_doc/dev-run.md:74]: "Start the MasterServer"

File: docs/en-us/dev/user_doc/dev-run.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+
+ Run the main method of org.apache.dolphinscheduler.server.master.MasterServer, you need to set the following VM options:
```
Comment (ptyp) [inline docs/en-us/dev/user_doc/dev-run.md:76]: ", you need" => ". You need"

File: docs/en-us/dev/user_doc/dev-run.md [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+
+```
+-Dlogging.config=classpath:logback-master.xml -Ddruid.mysql.usePingMethod=false
+```
+
+ix. Start WorkerServer
```
Comment (ptyp) [inline docs/en-us/dev/user_doc/dev-run.md:82]: "Start the WorkerServer"

File: docs/en-us/dev/user_doc/dev-run.md [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+
+ Run the main method of org.apache.dolphinscheduler.server.master.MasterServer, you need to set the following VM options:
+
+```
+-Dlogging.config=classpath:logback-master.xml -Ddruid.mysql.usePingMethod=false
+```
+
+ix. Start WorkerServer 
+
+run the main method of org.apache.dolphinscheduler.server.worker.WorkerServer, you need to set the following VM options:
```
Comment (ptyp) [inline docs/en-us/dev/user_doc/dev-run.md:84]: ", you need" => ". You need"

File: docs/en-us/dev/user_doc/dev-run.md [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+
+```
+-Dlogging.config=classpath:logback-worker.xml -Ddruid.mysql.usePingMethod=false
+```
+
+x. Start ApiApplicationServer
+
+run the main method of org.apache.dolphinscheduler.api.ApiApplicationServer, you need to set the following VM options:
```
Comment (ptyp) [inline docs/en-us/dev/user_doc/dev-run.md:92]: It should be the capital R in the beginning.
", you need" => ". You need"

File: docs/en-us/dev/user_doc/dev-run.md [hunk 11]
```diff
@@ -0,0 +1,135 @@
...
+
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | bash
+
+​	b. Refresh the environment variables
+
+​	source ~/.bash_profile
+
+​	c. Install node
+
+​	nvm install v12.20.2 note: Mac users could install npm through brew: brew install npm
```
Comment (ptyp) [inline docs/en-us/dev/user_doc/dev-run.md:116]: "note: ..." should be set in a new line I think.

**Review by ptyp — APPROVED** [review]

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**